### DISCUSSION TITLE
Now you can run PocketBase as an npm dependency: gobot 1.0.0-alpha.1

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#4459] [Show and tell]
Edit: pbgo has been renamed to `gobot` per #4497 

https://www.npmjs.com/package/gobot

```bash
npx gobot@latest pocketbase --help
``` 

`gobot` is an npm package that will download and run the latest version of popular statically-linked binaries including PocketBase. It considers your OS and CPU architecture and downloads the correct binary.

This package also has a programmatic API so you can add `gobot` as a dependency in projects and rest assured that the correct external binaries will always be available on the platform. Kind of inspired by `esbuild`.

You can also run specific versions of PocketBase. `gobot` will cache different versions of PocketBase and will check for new versions once per 24h.

We can use testers for various platforms, so please jump in and lend feedback!

**Comment by VictorioBerra** [comment]
@benallfree 

> This package has been deprecated
Author message:
Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.

**Reply by benallfree** [reply]
Apologies: #4497 

`npx gobot@latest pocketbase --help`
